packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### 5.0.0
```

Comment:
After the extensive testing, I wouldn't deem this a breaking change, as in both cases (`launchMode`s) it just works better.

So if testing from the reports in #162 is also fine, I would consider releasing this as just `4.3.0`.

What do you think @HenriBeck ?

Reply:
Should we add the docs also the the README?

Reply:
Good point. I'll adapt the version number and add the `launchMode` info into the README for Android.